website/content/docs/configuration/worker/index.mdx

```diff
@@ -74,6 +74,9 @@ worker {
   using env or file, their contents must formatted as a JSON array:
   `["127.0.0.1", "192.168.0.1", "10.0.0.1"]`
 
+  HCP Boundary workers require the [`hcp_boundary_cluster_id`](/boundary/docs/configuration/worker/#hcp_boundary_cluster_id) parameter instead of `initial upstreams`.
+  If you configure an HCP worker with `initial_upstreams`, the worker configuration fails.
+
```

Comment:
Looks great!

I'm wondering if we should move the section below on hcp_boundary_cluster_id to come before tags, and if we should change the wording in that section to remove the "can" so it doesn't sound optional

Reply:
Yes! I like both ideas. Good catch on the wording for `hcp_boundary_cluster_id`. I just added those updates.